portfolio/src/main/webapp/script.js

```diff
@@ -11,18 +11,3 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
```

Comment:
No javascript?

Reply:
Old comment, disregard.

(i see you're bringing in libraries to handle most everything.